external-effects/super-blur/SuperBlur.h

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file
+ * @brief Header file for SuperBlur effect class
+ * @author Jonathan Thomas <jonathan@openshot.org>
+ *
+ * @ref License
+ */
+
+/* LICENSE
+ *
+ * Copyright (c) 2008-2019 OpenShot Studios, LLC
+ * <http://www.openshotstudios.com/>. This file is part of
+ * OpenShot Library (libopenshot), an open-source project dedicated to
+ * delivering high quality video editing and animation solutions to the
+ * world. For more information visit <http://www.openshot.org/>.
+ *
+ * OpenShot Library (libopenshot) is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * OpenShot Library (libopenshot) is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPENSHOT_EXTERNAL_BLUR_EFFECT_H
+#define OPENSHOT_EXTERNAL_BLUR_EFFECT_H
+
+#include "../../include/EffectBase.h"
+
+#include <cmath>
+#include <ctime>
+#include <iostream>
+#include <omp.h>
+#include <stdio.h>
+#include <memory>
+#include <QRect>
+#include "../../include/Color.h"
+#include "../../include/Exceptions.h"
+#include "../../include/Json.h"
+#include "../../include/KeyFrame.h"
+#include "../../include/ReaderBase.h"
+#include "../../include/FFmpegReader.h"
+#include "../../include/QtImageReader.h"
+#include "../../include/ChunkReader.h"
+
+using namespace std;
```

Comment:
Remove this, please, and prefix all namespaced symbols in header files. See #296 .

Reply:
Done